sim/pokemon.js

```diff
@@ -1152,18 +1152,12 @@ class Pokemon {
 	}
 
 	/**
-	 * @param {string} itemName
 	 * @param {Pokemon} source
-	 * @param {Effect} sourceEffect
 	 */
-	useItem(itemName, source, sourceEffect) {
+	useItem(itemName, source) {
```

Comment:
Oh, you forgot to remove `itemName` here, it looks like you did intend to remove it.

Reply:
Yeah, I managed to remove the typescript for it, but not the parameter itself, oops...